hieradata/node/lsstcam-rebps.cp.lsst.org.yaml

```diff
@@ -0,0 +1,32 @@
+---
+nm::connections:
+  enp0s31f6:
+    content:
+      connection:
+        id: "enp0s31f6"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "enp0s31f6"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.175.78/26,139.229.175.126"
+        dns: "139.229.160.53;139.229.160.54;139.229.160.55;"
+        dns-search: "cp.lsst.org;"
+        method: "manual"
+      ipv6:
+        method: "disabled"
+      proxy: {}
+  enp3s0:
+    content:
+      connection:
+        id: "enp3s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp3s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.121/24"
```

Comment:
This IP also doesn't appear to be associated with a host.

Reply:
Hi @jhoblitt  what do you mean by this ? 🤔

Reply:
Done.